source/ios/tools/AdaptiveCards.podspec

```diff
@@ -12,32 +12,36 @@ Pod::Spec.new do |spec|
   
   spec.summary          = 'Adaptive Cards are a new way for developers to exchange card content in a common and consistent way'
   
-  spec.source       = { :git => 'https://github.com/microsoft/AdaptiveCards.git', :tag => 'ios-v2.8.1' }
+  spec.source       = { :git => 'https://github.com/microsoft/AdaptiveCards.git', :tag => 'ios-v2.8.2' }
```

Comment:
Can you create another PR to do the version number change?
It will be easy for non iOS developer to know that if I bump up the version number, I need to change these files.

Reply:
@licanhua, 
sure.
i'm updating ios release pipeline, and the pipeline will update this file and commit it individually and automatically.
for this pr, could we leave it as it is?